cmd/frontend/internal/cli/http.go

```diff
@@ -51,9 +54,20 @@ func newExternalHTTPHandler(
 	// immediately delegates the request to the next middleware in the chain).
 	authMiddlewares := auth.AuthMiddleware()
 
+	gh := webhooks.GitHubWebhook{
+		DB: db,
+	}
```

Comment:
Critical: remove 5 bytes

```suggestion
	gh := webhooks.GitHubWebhook{DB: db}
```

Reply:
Sometimes I'm slightly annoyed that the `go fmt` tool doesn't do these things automatically